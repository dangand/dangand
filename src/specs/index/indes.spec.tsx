import Index from "@/app/page";
import { render } from "@testing-library/react";

describe("Index", () => {
  it("renders successfully", () => {
    expect(() => {
      render(<Index />);
    }).not.toThrow();
  });
});

// Snapshot test
describe("Index", () => {
  it("renders successfully", () => {
    const { baseElement } = render(<Index />);

    expect(baseElement).toMatchSnapshot();
  });
});
