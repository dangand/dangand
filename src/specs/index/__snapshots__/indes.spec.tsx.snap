// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders successfully 1`] = `
<body>
  <div>
    <main
      class="flex flex-col items-center justify-between min-h-screen p-24"
    >
      <p>
        Hello
      </p>
    </main>
  </div>
</body>
`;
